public/app/features/live/centrifuge/serviceWorkerProxy.ts

```diff
@@ -28,6 +28,14 @@ export class CentrifugeServiceWorkerProxy implements CentrifugeSrv {
     );
   };
 
+  /**
+   * Query over websocket
+   */
+  getQueryData: CentrifugeSrv['getQueryData'] = async (options) => {
+    const optionsAsPlainSerializableObject = JSON.parse(JSON.stringify(options));
```

Comment:
https://github.com/grafana/grafana/pull/41653#issuecomment-982825811 that was the issue. JSON parse+stringify is one option, the other one is just explicitly pick the properties we need and make sure they are plain&serializable @ryantxu

Reply:
Is the main offender is timeRange?  

I think making the object more explicit is a better plan -- and exclude moment!